<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>916</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="sideMenuContainer">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
border: none;
background-color:rgb(32, 34, 37);
border-top-left-radius: 10px;
border-top-right-radius: 0px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 0px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>261</width>
            <height>581</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTreeWidget::item{
    color: rgb(255, 255, 255);
}

QTreeWidget::item:selected{
    background-color: rgb(173, 173, 173);
    color: rgb(0, 0, 0);
}

QHeaderView::section {
    background-color: rgb(32, 34, 37);
	border: none;
    color: white;
}</string>
          </property>
          <column>
           <property name="text">
            <string>Wood Service</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Forest Wood</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>A15 Invoice No</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A16 Forest Wood DO No</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A17 Lorry Transport</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A18 Convective M3 Tons</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Rubber Wood</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>A19 Finger joined KD Charge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A20 KD Handling Charges Master Data Entry</string>
            </property>
            <property name="toolTip">
             <string>A20 KD Handling Charges Master Data Entry</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A21 Rubber Wood DO No</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A22 Container Loading Charges Master Data Entry</string>
            </property>
           </item>
          </item>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">QFrame{
border-radius: 0px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="header">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(47, 49, 54);
border-top-right-radius: 10px;
}
QPushButton {
color: rgb(0, 0, 0);
padding: 0px;
border-radius: 15px;
border-right: none;
border-left: none;
}
QPushButton:hover{
background-color: rgb(90, 90, 90);
border-color: rgb(85, 0, 255);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="styleSheet">
                <string notr="true">QFrame{
border-radius: 0px;
}
QPushButton{
border: none;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QFrame" name="frame_6">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>60</width>
                  <height>60</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="sideMenuBtn">
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
border-radius: 0px;
}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../icons.qrc">
                     <normaloff>:/newPrefix/icons/menu-bar-white.png</normaloff>:/newPrefix/icons/menu-bar-white.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QLabel" name="widgetLabel">
                <property name="geometry">
                 <rect>
                  <x>80</x>
                  <y>0</y>
                  <width>201</width>
                  <height>60</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;</string>
                </property>
                <property name="text">
                 <string>Wood Service</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="styleSheet">
                <string notr="true">QFrame{
border-radius: 0px;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
border-radius: 10px;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>20</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="minBtn">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../icons.qrc">
                    <normaloff>:/newPrefix/icons/minimize-white.png</normaloff>:/newPrefix/icons/minimize-white.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="maxBtn">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../icons.qrc">
                    <normaloff>:/newPrefix/icons/maximize-white.png</normaloff>:/newPrefix/icons/maximize-white.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="closeBtn">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:hover{
background-color: rgb(200, 0, 0);
border-color: rgb(85, 0, 255);
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../icons.qrc">
                    <normaloff>:/newPrefix/icons/close-whitee.png</normaloff>:/newPrefix/icons/close-whitee.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>21</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="mainWidget">
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 10px;
}

QStackedWidget{
border-radius: 0px;
}


QPushButton {
	color: rgb(0, 0, 0);
	border-style: outset;
	padding: 0px;
	border-width: 1px;
	border-radius: 0px;
	border-color: rgb(122, 122, 122);
	background-color: rgb(245, 245, 245);
}
QPushButton:hover{
	background-color: rgb(180, 255, 255);
	border-color: rgb(85, 0, 255);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}


QLineEdit {
    border: 1px solid gray;
	background-color: white;
}

QTableView{
	border: 1px solid gray;
	border-radius: 10px;
}

/*
QTableView::item {
	border: 1px dashed gray; 
	padding: 5px;
}
*/

QHeaderView::section {
    background-color: rgb(236, 236, 236);
    color: black;
    padding-left: 4px;
    border: none;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <widget class="QFrame" name="sizeGripCorner">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(209, 209, 209)</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="sizeGripY">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QFrame" name="sizeGripX">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="styleSheet">
                <string notr="true">QWidget{
	border-radius: 0px;
}
QTableView{
	border-radius: 5px;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
